modules/dag-l1/src/main/scala/org/tessellation/dag/l1/Main.scala

```diff
@@ -23,7 +23,7 @@ import org.tessellation.sdk.resources.MkHttpServer.ServerName
 import com.monovore.decline.Opts
 import eu.timepit.refined.boolean.Or
 
-object Main extends TessellationIOApp[Run]("", "DAG L1 node", version = BuildInfo.version) {
+object Main extends TessellationIOApp[Run]("", "DAG L1 node", "L1", version = BuildInfo.version) {
```

Comment:
I think that if we want it to give some more realistic level of safety the `clusterId` should be a bit more convoluted than just "L1". I can imagine state channel owners calling their implementations as L1 as well. Maybe this could actually be autogenerated during build to some value? But then we would probably prevent joining between versions, which may not be desirable. Food for thought.

Reply:
I'd address that together with network deployment ticket

Reply:
On the other hand, we could validate `BuildInfo` hash as well so even if someone names his cluster `L1` then the version won't fit. But I get your point.